frontend/src/components/LinkInformations/RedeemSelfCreator.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="redeem-self-creator">
+    <redeem-information :user="user" :amount="amount" :memo="memo" />
+
+    <b-jumbotron>
+      <div class="mb-3 text-center">
+        <div class="mt-3">
+          {{ $t('gdd_per_link.no-redeem') }}
+          <a to="/transactions" href="#!">
+            <b>{{ $t('gdd_per_link.link-overview') }}</b>
+          </a>
+        </div>
+      </div>
+    </b-jumbotron>
+  </div>
+</template>
+<script>
+import RedeemInformation from '@/components/LinkInformations/RedeemInformation.vue'
+
+export default {
+  name: 'RedeemSelfCreator',
+  components: {
+    RedeemInformation,
+  },
+  props: {
+    user: { type: Object, required: true },
```

Comment:
just `username`?

Reply:
dieses user object wird über ein object mit v-bind übergeben. 
es kann also erst in der nächsten übergabe zum string werden 
ich reiche nun firstName weiter.